src/main/java/com/carpentersblocks/api/package-info.java

```diff
@@ -0,0 +1,4 @@
+@API(owner = "CarpentersBlocks", provides = "CarpentersBlocks|API", apiVersion = "3.3.7")
+package net.shadowfacts.yaw.api;
```

Comment:
Oh gawd, I'm really derpy today. I just copy/pasted the thing from mine and didn't realize I had to change the package >.<